labs/src/main/scala/org/scalalabs/basic/lab03/RecursionPatternMatchingExercise.scala

```diff
@@ -37,7 +41,17 @@ object RecursionPatternMatchingExercise {
    * List(1,1,2,3,1,1) -> List(1,1), List(2), List(3), List(1,1)
    */
   def groupConsecutive[T](in: List[T]): List[List[T]] = {
-    error("fix me")
+    in match {
+      case Nil => Nil :: Nil
```

Comment:
I believe `Nil :: Nil` is a list containing the empty list.
I believe `Nil` is just the empty list.

```
scala> Nil
res0: scala.collection.immutable.Nil.type = List()

scala> Nil :: Nil
res1: List[scala.collection.immutable.Nil.type] = List(List())
```